tools/system_test_jobs.groovy

```diff
@@ -8,7 +8,7 @@ DRIVERS = [ "nidcpower", "nidigital", "nidmm", "nifgen", "niscope", "niswitch",
 PLATFORMS = [ "win32", "win64" ]
 
 ROOT_FOLDER = "nimi-bot"
-credentials_to_use = 'c07926b1-d626-4476-892a-e21bb6d28733'  // nimi-bot username/Personal Access Token
+credentials_to_use = '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'  // nimi-bot username/Personal Access Token
```

Comment:
We manually copy this groovy script into Jenkins job configuration. You can leave this field empty and populate it when we copy to Jenkins. So, we're not putting the PAT out in the open.